src/App.jsx

```diff
@@ -14,33 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const sortGoods = (goods, sortType, isReverse) => {
+  let sortedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (sortType) {
+    case 'alphabet':
+      sortedGoods = [...goods].sort((a, b) => a.localeCompare(b));
```

Comment:
```suggestion
      sortedGoods.sort((a, b) => a.localeCompare(b));
```
You have already created a copy